Category-Colors/category-colors
# Install Chakra UI

# Description
It's a pretty standard install of ChakraUI using `npm i @chakra-ui/react @emotion/react @emotion/styled framer-motion`.

# QA
Go to `http://localhost:3000/` and you should see hello world on the page. 
![Screen Shot 2023-04-23 at 10 00 43 PM](https://user-images.githubusercontent.com/2035194/233882645-27e27e80-0cdc-4acc-9c8a-1dba59ce8782.png)
